tests/test_DataSet.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import os
+
+import RadClass.DataSet as ds
+import tests.create_file as file
```

Comment:
I find the naming a little challenging to parse. Maybe:
```suggestion
import tests.create_file as test_data
```

Reply:
It may also be true that `create_file` isn't the best file name since it contains info and also a method to `create_file`